### PR TITLE
Update documentation for proposed time format fix

### PR DESCRIPTION
**Pull request by seanvictory**
**Description:**
Adds documentation for the custom time formats

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26631

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase it to the `next` branch

File: source/_components/worldclock.markdown [hunk 1]
```diff
@@ -22,6 +22,7 @@ To enable this sensor in your installation, add the following to your `configura
 sensor:
   - platform: worldclock
     time_zone: America/New_York
+    time_format: "%-I:%M %p"
```
Comment (klaasnicolaas) [inline source/_components/worldclock.markdown]: Only required variables in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/worldclock.markdown (hunk 1 above)
Comment (seanvictory) [reply source/_components/worldclock.markdown]: Thanks for the info! Made the changes.

**Comment by klaasnicolaas** [comment]
There are still a lot of unrelated commits

**Comment by seanvictory** [comment]
I think all of those commits were in response to doing the rebase. Anything you'd recommend doing to fit it or should I just make a new PR and copy-paste my changes?

**Comment by klaasnicolaas** [comment]
Pull the latest changes from our upstream repo to your fork and then push back again to this PR.

File: source/_components/worldclock.markdown [hunk 2]
```diff
@@ -34,6 +34,11 @@ name:
   required: false
   type: string
   default: Worldclock Sensor
+time_format:
+  description: The format of the time
+  required: false
+  type: string
+  defaul: "%H:%M"
```
Comment (fabaff) [inline source/_components/worldclock.markdown]: It's `default`.

File: source/_components/worldclock.markdown [hunk 3]
```diff
@@ -34,6 +34,11 @@ name:
   required: false
   type: string
   default: Worldclock Sensor
+time_format:
+  description: The format of the time
```
Comment (fabaff) [inline source/_components/worldclock.markdown]: Period is missing.

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_components/worldclock.markdown (hunk 3 above)
Comment (fabaff) [reply source/_components/worldclock.markdown]: We should add a link for further details here (e.g., https://docs.python.org/3.8/library/datetime.html#strftime-strptime-behavior) as it's not obvious for a person who is not familiar with Python.

File: source/_components/worldclock.markdown (hunk 2 above)
Comment (seanvictory) [reply source/_components/worldclock.markdown]: Thank you!

File: source/_components/worldclock.markdown (hunk 3 above)
Comment (seanvictory) [reply source/_components/worldclock.markdown]: Good point, thank you

**Comment by frenck** [comment]
@seanvictory Could you please cleanup this PR by removing all the unrelated commits?

We can't merge this in the current state. Thanks! 👍

**Comment by frenck** [comment]
The fork is gone, so there is no fixing possible at this point.
Going to close this PR and mark the parent PR as `docs-missing`.

**Pull request closed without merging**